src/plugins/search/plugin.search.js

```diff
@@ -66,23 +66,28 @@ BookReader.prototype.setup = (function (super_) {
     /** @type { {[pageIndex: number]: SearchInsideMatchBox[]} } */
     this._searchBoxesByIndex = {};
 
-    this.searchView = undefined;
+    if (this.enableSearch) {
+      this.searchView = new SearchView({
+        br: this,
+        searchCancelledCallback: () => {
+          this._cancelSearch();
+          this.trigger('SearchCanceled', { term: this.searchTerm, instance: this });
+        }
```

Comment:
SUGGESTION: Add comma after `}`

Reply:
Will add to eslint to auto-fix/enforce and do in a new PR :+1: 

(thanks chatgpt! ![image](https://github.com/user-attachments/assets/0daee3a9-1403-4902-b05b-207b3342a2d4) )